docs/tapiro/migrate.md

```diff
@@ -0,0 +1,42 @@
+---
+id: migrate
+title: Migration from Wiro
+---
+
+Tapiro is meant to deprecated [wiro](https://github.com/buildo/wiro).
+
+Tapiro is based on the same concepts of wiro, the migration is pretty straightforward.
+
+Here is a checklist of what you need to do:
+1. Install the plugin (as described in the [guide](installation.md))
+2. Configure your `build.sbt` (as described in the [guide](installation.md))
+3. Add `AuthToken` type parameter to controllers
+    `trait AccountController` -> `trait AccountController[AuthToken]`
+4. Modify controllers so that wiro `Auth` is replaced with AuthToken and move as last argument
+    `def read(token: Auth, arg: Int)` -> `def read(arg: Int, token: AuthToken)`
+5. Add `.gitattributes` file containing `**/*Endpoints.scala linguist-generated` so that autogenerated code is hidden by default in git diffs
```

Comment:
I find this line hard to understand. I think it's better
> Add `**/*Endpoints.scala linguist-generated` to repository's `.gitattributes` to hide tapiro generated code from git diff's

Reply:
s/git/github